test/src/androidTest/kotlin/io/realm/InstrumentedTests.kt

```diff
@@ -72,6 +80,8 @@ class InstrumentedTests {
 
     @Test
     fun query() {
+        val realm = openRealmFromTmpDir()
```

Comment:
Seems like a common thing, so should probably go to the `setup` method.

Reply:
Good refactor next Sprint